content/docs/storage/platforms/powerstore/Product_Guide.md

```diff
@@ -0,0 +1,1593 @@
+---
+title: "Product Guide"
+linkTitle: "PowerStore"
+no_list: true
+description: Documentation for PowerStore
+weight: 1
+---
+
+# Terraform Modules for Dell PowerStore
+--------------
+# Contents
+*   [Volume Resource](#volume-resource)
+    *   [Synopsis](#synopsis)
+    *   [List Of Parameters](#list-of-parameters)
+    *   [Examples](#examples)
+    *   [Notes](#notes)
+*   [SnapshotRule Resource](#snapshotrule-resource)
+    *   [Synopsis](#synopsis-1)
+    *   [List Of Parameters](#list-of-parameters-1)
+    *   [Examples](#examples-1)
+    *   [Notes](#notes-1)
+*   [ProtectionPolicy Resource](#protectionpolicy-resource)
```

Comment:
I Think it should be Protection Policy and not ProtectionPolicy

Reply:
we have followed this naming convention everywhere. Confluence: https://confluence.cec.lab.emc.com/display/TFM/Resource+Schema+v1.0.0

Reply:
Ok . No Problem